edi_oca/README.rst

```diff
@@ -1,7 +1,3 @@
-.. image:: https://odoo-community.org/readme-banner-image
```

Comment:
tip: `export SKIP=oca-gen-addon-readme` to not touch readme files :wink:

Reply:
they just make noise and - worse part - when you cherry pick commits it's easy to get conflicts. Can you roll this back pls?